src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -134,20 +137,29 @@ private void GetDocuments(DocumentsOperationContext context, Transformer transfo
 
             using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
             {
+                if (isLoadStartingWith)
```

Comment:
This introduces inconsistency. Endpoints that are returning documents return them in different JSON format depending of the option used. We should not go this way in our opinion.